src/dotnet-interactive-vscode-insiders/README.md

```diff
@@ -2,31 +2,31 @@ This extension is currently **_in preview_**.
 _The Visual Studio Code notebook support that this extension uses is also in preview and design is ongoing, so the extension might not work._
 ---
 
-# .NET Interactive Notebooks
+# Polyglot Notebooks
 
-This extension adds support for using .NET Interactive in a Visual Studio Code notebook.
+This extension adds support for using Polyglot in a Visual Studio Code notebook.
 
 ## Getting Started
 
 1.  Install the latest [Visual Studio Code](https://code.visualstudio.com/).
 
 2.  Install the latest [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
 
-3.  Install the .NET Interactive Notebooks extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode).
+3.  Install the Polyglot Notebooks extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode).
 
 ## Opening and Creating notebooks
 
-To open an existing .NET notebook, bring up the Command Palette(`Ctrl+Shift+P`) and select **.NET Interactive: Open notebook**.  Now, navigate to a local `.ipynb` file.
+To open an existing .NET notebook, bring up the Command Palette(`Ctrl+Shift+P`) and select **Polyglot: Open notebook**.  Now, navigate to a local `.ipynb` file.
 
 ![opennotebook](https://user-images.githubusercontent.com/2546640/94441970-67d6e180-0171-11eb-8319-c12ba82c3d30.gif)
 
-To create a new notebook, open the Command Palette(`Ctrl+Shift+P`), and select **.NET Interactive: Create new blank notebook**. You can also create a new notebook with `Ctrl+Shift+Alt+N` key combination.
+To create a new notebook, open the Command Palette(`Ctrl+Shift+P`), and select **Polyglot: Create new blank notebook**. You can also create a new notebook with `Ctrl+Shift+Alt+N` key combination.
```

Comment:
```suggestion
To create a new notebook, open the Command Palette(`Ctrl+Shift+P`), and select **Polyglot  Notebook: Create new blank notebook**. You can also create a new notebook with `Ctrl+Shift+Alt+N` key combination.
```

Reply:
I think "Polyglot Notebook" reads better here but I believe the extension name would be "Polyglot Notebooks". I'm assuming we should align these. 

Thoughts, @claudiaregio?

Reply:
I agree with you Jon that "Polyglot Notebook" reads better and there is prior art for the command prefix and extensions not matching perfectly. There is the GitHub Issue Notebooks extension but the command is "New GitHub Issue Notebook". This was the same question I was asking in the first chat too basically. 

If we can have the extension be "Polyglot Notebooks" but the commands prefixed with "Polyglot Notebook" I think that would make most sense.